src/utilities/theme/tokens.ts

```diff
@@ -38,7 +38,7 @@ export const Tokens = {
   textFieldSpinnerOffset: rem('2px'),
   textFieldFocusRingOffset: rem('-4px'),
   textFieldFocusRingBorderRadius: rem('7px'),
-  buttonGroupItemSpacing: rem('2px'),
+  buttonGroupItemSpacing: rem('-1px'),
```

Comment:
As we are no longer using the `--p-button-group-item-spacing` variable maybe we should remove it?

Reply:
Oh we were when we were pairing. Ashwin it looks like the connected filters changes didn't get pushed up

Reply:
There were no changes for the ConnectedFilterControl component. The --p-button-group-item-spacing variable is still being used in the component. It is still necessary because it's used as a fallback (without the new design language).

Reply:
Thanks for the context!